packages/core/src/messages/responses/enums/index.ts

```diff
@@ -0,0 +1,28 @@
+export enum MomentoResponse {
+  Success = 'Success',
+  Error = 'Error',
+}
+
+export enum CacheGetResponse {
```

Comment:
this is a bit weird, maybe we can just have like a `CacheFetchResponse` for generic hit, miss, error responses